assets/src/scss/layout/_navbar.scss

```diff
@@ -75,6 +74,16 @@ $navbar-default-height: 60px;
       top: 32px;
     }
   }
+
+  a, .btn, input {
+    &:focus {
+      --top-navigation--ui-controls-- {
+        outline: 2px solid rgba(255, 255, 255, 0.4);
+      }
+      // Need to undo the bootstrap outline-offset to get a consistent result.
+      outline-offset: 0;
+    }
+  }
```

Comment:
I made this into a single declaration, so that they can all be changed in one go if the top navigation color changes.